tests/unit_tests/pyanaconda_tests/modules/payloads/source/test_source_live_os.py

```diff
@@ -178,8 +199,12 @@ def test_detect_live_os_image(self, os_stat_mock, stat_mock, exists_mock, exec_m
         detected_image = task.run()
         assert detected_image == "/my/device"
 
-    def test_setup_install_source_task_name(self):
+    @patch("pyanaconda.modules.payloads.source.live_os.initialization.SetUpLiveOSSourceTask"
+           "._calculate_required_space")
```

Comment:
I think you can use `@patch.object(SetUpLiveOSSourceTask, "_calculate_required_space", return_value=0)` to make it simpler. Then you don't have to call `required_space.return_value = 0`.

Reply:
Maybe use a different size than zero in these tests. You are trying to simulate the real usage.